js/build.js

```diff
@@ -148,8 +148,8 @@
             }));
 
             if (data.entries.length) {
-              data.entries[0].sliced = true;
-              data.entries[0].selected = true;
+              data.entries[0].sliced = false;
```

Comment:
Why does this need an assignment to `false` if the default values are `false` or `undefined`?